tests/unit/metadata/test_table.py

```diff
@@ -582,93 +581,6 @@ def test__transform_constraints_drops_columns(self):
         }, index=[0, 1, 2])
         assert result.equals(expected_result)
 
-    def test__validate_data_on_constraints(self):
```

Comment:
Can we keep the tests for this method but just move them to the appropriate place